_haikus/Cloud Services.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Cloud Services
+author: Vishnusai6298
+---
+
+Cloud Providers give<br>
+Services that are broadly<br>
+SaaS Paas IaaS<br>
```

Comment:
There aren't enough syllables here, usually `SaaS` is pronounced "sass", `PaaS` is pronounced "pass", and `IaaS` is pronounced "ice", so there would only be 3 syllables here. Please add two more.